src/Mod/PartDesign/Gui/Command.cpp

```diff
@@ -664,6 +664,115 @@ unsigned validateSketches(std::vector<App::DocumentObject*>& sketches,
     return freeSketches;
 }
 
+/**
+ *  Partially pulled from Linkstage3 importExternalObjects for toponaming element map
+ *  compatibility with sketches that contain point objects.  By adding an empty
+ *  subobject when appropriate, we allow those sketches to be used as profiles without error.
+ *
+ * @param prop  The property ( generally a Profile link )
+ * @param _sobjs    Subobjects to use
+ * @param report    True if we should raise a dialog, otherwise raise and exception
+ * @return  True if elements were found
+ */
+bool importExternalElements(App::PropertyLinkSub& prop,
+                             std::vector<App::SubObjectT> _sobjs,
+                             bool report)
```

Comment:
Do we really need that "report" boolean? I'd just strip out the entire try block and let the exception propagate.

Reply:
Totally agree.  I stripped a lot, but wasn't positive on this one.  It'll be gone shortly.